Proofs/SHA512/Sha512_block_armv8.lean

```diff
@@ -80,6 +81,25 @@ theorem sha512_program_test_2_sym (s0 s_final : ArmState)
 
 -- Test 3:
 
+theorem Aligned_BitVecAdd_64_4 {x : BitVec 64} {y : BitVec 64}
+  (x_aligned : Aligned x 4)
+  (y_aligned : Aligned y 4)
+  :
+  Aligned (x + y) 4 := by
+  unfold Aligned at *
+  simp_all
+  bv_decide
```

Comment:
```suggestion
theorem Aligned_BitVecAdd_64_4 {x : BitVec 64} {y : BitVec 64}
  (x_aligned : Aligned x 4)
  (y_aligned : Aligned y 4)
  :
  Aligned (x + y) 4 := by
  simp_all [Aligned]
  bv_decide
```

Golf by folding the `unfold` into the `simp_all`.

Reply:
Nice catch!